### PR TITLE
test(rate): Update this rate component unit test.

### LINKED ISSUE #547: [Test] Improve unit testing of rate components

### PR DESCRIPTION
**Pull request by wegi8**
## Checklist

---

- [ ] Fix linting errors
- [x] Tests have been added / updated (or snapshots)

## Change information

---
close #547

### CODE REVIEW

**Comment by songjianet** [comment]
The component name of the title is misspelled.

**Comment by wegi8** [comment]
> The component name of the title is misspelled.

I corrected it.

File: packages/varlet-ui/src/rate/__tests__/index.spec.js [hunk 1]
```diff
@@ -72,64 +43,83 @@ test('test rate validation', async () => {
...
-  expect(wrapper.html()).toMatchSnapshot()
+  test('test rate gap', () => {
+    const wrapper = mount(VarRate, {
+      props: {
+        gap: 10,
+      },
+    })
 
-  wrapper.unmount()
+    console.log('wrapper.element', wrapper.element)
```
Comment (BeADre) [inline packages/varlet-ui/src/rate/__tests__/index.spec.js:119]: It seems this test case has not been completed.

Comment (wegi8) [reply packages/varlet-ui/src/rate/__tests__/index.spec.js:119]: I don't know how to compare its `gap` correctly.

Comment (BeADre) [reply packages/varlet-ui/src/rate/__tests__/index.spec.js:119]: The `gap` prop actually adds `margin-right` for `var-rate__content`, so we can test it like this:

```
const el = wrapper.find('.var-rate__content')

expect(el.attributes('style')).toMatch('margin-right: 10px')
```

Comment (wegi8) [reply packages/varlet-ui/src/rate/__tests__/index.spec.js]: it's work.But i'm not sure the Reg is Conforming to the specification

**Review by BeADre — APPROVED** [review]

**Pull request merged**